tests/pkg_fatfs_vfs/tests/01-run.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
with `#!/usr/bin/env python` it fails, with `#!/usr/bin/env python3` it works (for me ;)